apps/src/gamelab/spritelab/Spritelab.js

```diff
@@ -0,0 +1,218 @@
+  };
+
+  this.commands.moveForward = function(spriteId, distance) {
+    commands.moveForward(spriteId, distance);
+  };
+
+  this.commands.moveToward = function(spriteId, distance, target) {
+    commands.moveToward(spriteId, distance, target);
+  };
+
+  this.commands.removeTint = function(spriteId) {
+    commands.setProp(spriteId, 'tint', null);
+  };
+
+  this.commands.setProp = function(spriteId, prop, val) {
+    commands.setProp.apply(this, [spriteId, prop, val]);
+  };
+
+  this.commands.setTint = function(spriteId, color) {
+    commands.setProp(spriteId, 'tint', color);
+  };
+
+  this.commands.turn = function(spriteId, n, direction) {
+    commands.turn(spriteId, n, direction);
+  };
+
+  // Behavior commands
+  this.commands.addBehaviorSimple = function(spriteId, behavior) {
+    commands.addBehavior(spriteId, behavior);
+  };
+
+  this.commands.Behavior = function(callback) {
+    return commands.Behavior(callback);
+  };
+
+  this.commands.draggableFunc = function(spriteId) {
+    return commands.draggableFunc(this);
+  };
+
+  this.commands.removeAllBehaviors = function(spriteId) {
+    commands.removeAllBehaviors(spriteId);
+  };
+
+  this.commands.removeBehaviorSimple = function(spriteId, behavior) {
+    commands.removeBehavior(spriteId, behavior);
+  };
+
+  // Event commands
+  this.commands.checkTouching = function(
+    condition,
+    sprite1,
+    sprite2,
+    callback
+  ) {
+    commands.checkTouching(condition, sprite1, sprite2, callback);
+  };
+
+  this.commands.keyPressed = function(condition, key, callback) {
+    commands.keyPressed(condition, key, callback);
+  };
+
+  this.commands.spriteClicked = function(condition, spriteId, callback) {
+    commands.spriteClicked(condition, spriteId, callback);
+  };
+
+  // Location commands
+  this.commands.locationAt = function(x, y) {
+    return commands.locationAt(x, y);
+  };
+
+  this.commands.locationMouse = function() {
+    return commands.locationMouse.apply(this);
+  };
+
+  this.commands.locationOf = function(spriteId) {
+    return commands.locationOf(spriteId);
+  };
+
+  this.commands.randomLocation = function() {
+    return commands.randomLocation();
+  };
+
+  // Sprite commands
+  this.commands.countByAnimation = function(animation) {
+    return commands.countByAnimation(animation);
+  };
+
+  this.commands.createNewSprite = function(name, animation, location) {
+    return commands.makeSprite.apply(this, [animation, location]);
+  };
+
+  this.commands.destroy = function(spriteId) {
+    commands.destroy(spriteId);
+  };
+
+  this.commands.displace = function(spriteId, targetSpriteIndex) {
+    commands.displace(spriteId, targetSpriteIndex);
+  };
+
+  this.commands.getProp = function(spriteId, prop) {
+    return commands.getProp(spriteId, prop);
+  };
+
+  this.commands.getThisSprite = function(which, extraArgs) {
+    return commands.getThisSprite(which, extraArgs);
+  };
+
+  this.commands.makeNewSpriteAnon = function(animation, location) {
```

Comment:
Is this short for Anonymous? Can we just swap it to 'makeNewSpriteAnonymous' for clarity?

Reply:
Unfortunately, we can't easily change the function name, because that would require changing the block name as well. Changing the block name would break every project that already exists with the `makeNewSpriteAnon` block (which we definitely don't want to do haha)

+1 to your point, though, and definitely something to keep in mind when naming new blocks in the future.

Reply:
👍